ios/GoodDollar-Bridging-Header.h

```diff
@@ -0,0 +1,4 @@
+//
```

Comment:
and this file

Reply:
this file needs to make ios animations work.
@n make this the same. please see https://github.com/nicastelo/lottieTest